handbook/product/onboarding/index.md

```diff
@@ -1,72 +1,115 @@
-- Get to know the product
-  - Complete the "Getting set up" section of the [Engineering onboarding tasks](../../engineering/onboarding/index.md#getting-set-up).
-    - You will need to run Sourcegraph locally to test/validate work that engineering is doing, to provide early/often feedback.
-  - Review [product resources](../index.md#resources)
-  - [Products](https://about.sourcegraph.com/product)
-  - Learn how the Customer Engineering team gives demos and talks about the product in the [product demo recording](https://drive.google.com/file/d/1idbCnce5MIvtAV0GOOwgB68zQJB2WmZ9/view).
-  - Read about [search queries](https://docs.sourcegraph.com/user/search) and perform your first searches.
-  - Work through the questions from the [Sales Onboarding Quiz](../../sales/onboarding/quiz.md) to make sure you understand key concepts. Feel free to skip any obvious answers and discuss any questions you have or knowledge gaps with your manager.
-- Get to know our customers
-  - Reach out to the Sales/CE teams and ask to be added to as many customer calls as you can this week.
-  - Feedback
-    - [HubSpot](https://app.hubspot.com/forms/2762526/a86bbac5-576d-4ca0-86c1-0c60837c3eab/submissions)
-    - [Productboard](https://sourcegraph.productboard.com/insights/shared-inbox)
-- Understand our goals
-  - [Sourcegraph master plan](../../../company/strategy.md)
-  - [Sourcegraph direction (1 year plan)](../../../direction/index.md)
-  - [Goals](../../../company/goals/index.md)
-
-## Product designer onboarding
-
-After completing the product team checklist above, use the following resources to get up to speed on design at Sourcegraph.
+- Schedule daily check-ins for the first week at Sourcegraph to keep up with onboarding and to create space for answering any questions that might come up.
+
+## Week 1 - Getting started
+
+Your objective is to get to know the team and learn as much about Sourcegraph (the company and product) as possible. It is your responsibility to be proactive in your onboarding, and it is the team's #1 priority to support you and help set you up for success. Everyone will drop what they're doing to help you. We are so excited to have you on the team and can't wait to get to know you better!
+
+Remember:
+
+- Sourcegraph is an [open company](../../../company/open_source_open_company.md). Everything here at Sourcegraph is public unless there is a very good reason for it to be private.
+   - Good reasons for things to be private: sensitive customer information, sensitive personal information
+   - If you see a "request permission" page for some doc or other resource, it is a mistake. Just ask to be granted access to it (and mention it to Noemi so the next person starting is granted access to that system).
+- Unlike at many companies, it is OK (and important) to call out when we did something poorly or when something doesn't go well (e.g., a sales pitch falls flat). That helps us be real and do it better in the future.
+
+Onboarding buddy: TBD
```

Comment:
```suggestion
[Onboarding buddy](https://about.sourcegraph.com/handbook/people-ops/buddy-program): You will be paired up with an “onboarding buddy” in order to become better acquainted with Sourcegraph team and culture. A buddy will be your go-to person for questions (in addition to your manager and the People Ops team).
```

Not sure if we want to include this or if it was just pasted from the template, but if so, I'd add a little context and the link to the Onboarding Buddy page in the handbook